content/blog/2022-05-20-long-term-funding-support-for-etc-launches/index.en.md

```diff
@@ -0,0 +1,40 @@
+
+
+## [Ethereum Classic History](https://ethereumclassic.org/knowledge/history)
+
+![Ethereum Classic is Money](./etc-is-money.png)
+
+
+## Long Term Funding Solution for Development on ETC
+
+A move to erect a funding mechanism was signalled on GitCoin expressing the goal of future-proofing the network's development ecosystem. This news comes at an opportune time as Ethereum Classic's more popular, sister chain moves to Proof-of-Stake consensus. The result of this move is an abundance of computer equipment available to provide hashrate to secure Ethereum Classic, the main branch of the EVM ecosystem.
+
+
+## Ethereum Classic DAO & DeFi Stack - LTS Solution
+
+>LTS Funding Pool:
+>
+>The LTS funding pool is oriented in a way which will grow to become self-sustaining via the mechanics of the positive feedback loop. As funds are collected from one protocol, another protocol will be developed that amplifies the protocol stack. In turn, the fund will grow at a higher rate enabling more protocols to turn online at a quicker rate. At some point, the fund will begin to grow at a rate the is completely self sustaining to the needs of the network. Therefore the immediate needs of the network will be addressed. This will begin the fund down a new path where it is accumulating excess funds. This will transition the funds priorities from organizing the "needs" of the network to addressing the "wants" of the network as the outlook of develop moves to a more future driven approach. As this point, it is our intention to start to explore ways in which we can start addressing decentralized ways to allocate portions of the funding pool. We are not sure what will work, but our intention is to allocate small portions of the funding pool to specific tasks and attempt different governance methods. Through this low risk approach, we expect to see governance methods completely fail and others show promise. Governance in a decentralized way is one of the biggest challenges in the blockchain space, so we will stay open minded and continue to try to setup multiple methods. All of these will be with the intention of decentralizing the funding process through MANY channels of distribution and methods. We believe this approach reduces risk to the overall fund and builds redundancies in the funding channels. We understand this aspect of the project is very lofty and look forward to open involvement from the ETC community as we explore this goal.
```

Comment:
"wants" of the network as the outlook of develop moves to a more future driven approach.

development I'm assuming. :)

ronin, you've been busy. Wish you great success on your journey. Thank you for your efforts securing Ethereum Classic.🙏

Reply:
Thanks for the typo catch. :+1: Resolved.